lucene/core/src/java/org/apache/lucene/index/DocumentsWriterPerThread.java

```diff
@@ -351,7 +354,7 @@ FlushedSegment flush(DocumentsWriter.FlushNotifications flushNotifications) thro
         flushState.liveDocs.clear(deleteDocIDs[i]);
       }
       flushState.delCountOnFlush = numDeletedDocIds;
-      bytesUsed.addAndGet(-(deleteDocIDs.length * Integer.SIZE));
+      bytesUsed.addAndGet(-(deleteDocIDs.length * Integer.BYTES));
```

Comment:
Whoa, this was a latent (over-counting) bug?  Maybe `grep` for other places we are possibly incorrectly using `Integer.SIZE`!

Though, I think this array is typically tiny (usually 0 length), since it holds docs that hit non-aborting exception during indexing?

Reply:
yeah I checked all instanced in our codebase after I found this.